cli/generate.js

```diff
@@ -60,15 +72,14 @@ function generateComponentTs(pathParts, fileName, name, nameSnakeCase) {
 export class ${name} {
 
 }
-`
-  );
-}
+`;
 
-function generateComponentSpec(pathParts, fileName, name, nameSnakeCase) {
-  let nameWithSpaces = properCase(nameSnakeCase.replace(/\-/g, ' '));
+  safeFileWrite(pathParts, fileName + '.ts', fileContent, force);
```

Comment:
Non-blocking, but my branch that adopts eslint will throw an error for this type of string concatenation. It prefers the template:
```
`${fileName}.ts`
```

Reply:
I went ahead and updated to that style.  :-)